### PR TITLE
Fixed failures in sitesmap and assembly and module content

### PR DESCRIPTION
**Pull request by noopurmath3**

### CODE REVIEW

File: suites/test_module_edit_publish.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def publish_module(self, api_auth):
     response = api_auth.get(self.request_url)
     # Check that the node has been marked as released
     check_that("The published module now has a 'released' node", response.json()["en_US"]["variants"][self.variant],
-               contains_string("released"))
+               has_entry("released"))
```
Comment (anarang) [inline suites/test_module_edit_publish.py:86]: @noopurmath3 what was your finding that "contains string" was failing but "has_entry" would make it pass?

**Review by anarang — APPROVED** [review]
changes look straightforward, but I would be interested to know the reasoning.

File: suites/test_module_edit_publish.py (hunk 1 above)
Comment (noopurmath3) [reply suites/test_module_edit_publish.py:86]: @anarang yes its working with has_entry() and I think has_entry() is better suited here than contains_string()

**Review by pbadak — APPROVED** [review]
LGTM :+1:

**Pull request merged**